index.html

```diff
@@ -8,29 +8,180 @@
-        <a><h3 class="logo">My Logo</h3></a>
-        <nav>
-            <div class="nav-right">
-                <img src="src/img/Union.png" alt="des-menu" class="menu">
-            </div>
-        </nav>
-    </header>
-    <section class="main-container">
-        <div class="greeting-text">
-            <h1 class="name">I'm Jhonatan.<br>Glad to see you!</h1>
-            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+    <div class="mobile-main">
+        <header class="flex-div">
+            <a><h3 class="logo">My Logo</h3></a>
+            <nav>
+                <div class="nav-right">
+                    <img src="src/img/Union.png" alt="des-menu" class="menu">
+                </div>
+            </nav>
+        </header>
+        <section class="main-container">
+            <div class="greeting-text">
+                <h1 class="name">I'm Jhonatan.<br>Glad to see you!</h1>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            </div>
+            <div class="connect">
+                <p>LET'S CONNECT</p>
+            </div>
+            <ul class="media-icon">
+                <li><a href="" class="icon1"><img src="src/img/icons/Vector.png" alt="Twitter"></a></li>
+                <li><a href="" class="icon2"><img src="src/img/icons/Linkedin icon.png" alt="Linkedin"></a></li>
+                <li><a href="" class="icon3"><img src="src/img/icons/Vector (1).png" alt="icon3"></a></li>
+                <li><a href="" class="icon4"><img src="src/img/icons/Vector (2).png" alt="icon4"></a></li>
+                <li><a href="" class="icon5"><img src="src/img/icons/Vector (3).png" alt="icon5"></a></li> 
+            </ul>
+        </section>
+    </div>
+    <div class="works-container">
+        <div class="card-container">
+            <header class="thumbnail">
+                <img src="src/img/Snapshoot.png" alt="tonic1">
+            </header>
+            <div class="card">
+                <h2>Tonic</h2>
+                <ul class="info-card">
+                    <li class="canopy"><h3>CANOPY</h3></li>
+                    <li class="c-text"><h4> Back End Dev</h4></li>
+                    <li class="c-text"><h4> 2015</h5></li>
+                </ul>
+                <div class="card-text">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="languages">
+                    <a href="" class="btn">HTML</a>
+                    <a href="" class="btn">CSS</a>
+                    <a href="" class="btn">JavaScript</a>
+                </div>
```

Comment:
- [ ]  As the project requirements, please use `ul` and `li` for categories in "Work" section. Html, CSS, and JavaScript should be list items. :+1: